pdr_backend/lake/gql_data_factory.py

```diff
@@ -373,14 +373,18 @@ def _update(self):
 
             # fetch from subgraph and add to temp table
             logger.info("Updating table %s", get_table_name(table.table_name))
-            self._do_subgraph_fetch(
-                table,
-                self.record_config["fetch_functions"][table.table_name],
-                self.ppss.web3_pp.network,
-                st_ut,
-                fin_ut,
-                self.record_config["config"],
-            )
+            try:
+                self._do_subgraph_fetch(
+                    table,
+                    self.record_config["fetch_functions"][table.table_name],
+                    self.ppss.web3_pp.network,
+                    st_ut,
+                    fin_ut,
+                    self.record_config["config"],
+                )
+            except Exception as e:
+                self._move_from_temp_tables_to_live()
+                logger.error("Error on fetching data from %s: %s", table.table_name, e)
```

Comment:
I think we need to review this...

Reply:
removed that part after adding it, realised we don't want this